tests/test_process_elife_zip_dag.py

```diff
@@ -205,10 +198,26 @@ def test_strip_related_article_tags_from_article_xml_using_article_without_relat
     add_return_value_from_previous_task(article_xml, context=context)
 
     # test
-    return_value = strip_related_article_tags_from_article_xml(**context)
+    return_value = pezd.strip_related_article_tags_from_article_xml(**context)
     assert return_value == article_xml
 
 
+def test_add_missing_uri_schemes(context):
+    # setup
+    test_asset_path = str(get_asset('elife-36842.xml').absolute())
+    article_xml = etree.parse(test_asset_path)
+    xpath = '//*[starts-with(@xlink:href, "www.")]'
```

Comment:
`www.` seems very specific, but if it works for our examples it's good enough

Reply:
follow the bot as an example: https://github.com/elifesciences/bot-lax-adaptor/blob/58318286ab8d07d206abe34b313f63ce51117d31/src/main.py#L351-L370